keyboards/vortex/m0110a/1800_iso/info.json

```diff
@@ -0,0 +1,138 @@
+{
+    "manufacturer": "Vortex",
+    "keyboard_name": "M0110A-1800_ISO",
+    "maintainer": "Gsky",
+    "bootloader": "wb32-dfu",
+    "bootmagic": {
+        "matrix": [1, 6]
+    },
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "layer_count": 16
+    },
+    "features": {
+        "audio": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "num_lock": "C14",
+        "caps_lock": "C15"
+    },
+    "matrix_pins": {
+        "cols": ["A0", "A1", "A2", "A3", "A4", "A8", "A9", "A10"],
+        "rows": ["B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "B10", "B11", "B12", "B13"]
+    },
+    "processor": "WB32FQ95",
+    "qmk": {
+        "tap_keycode_delay": 10
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.2",
+        "pid": "0xE3B0",
+        "vid": "0x342D"
```

Comment:
https://www.the-sz.com/products/usbid/index.php?v=0x342D&p=&n=

This is using WB's vendor ID, but doesn't appear to be a board that they are releasing.  Please assign a different vendor ID.  The further past 0x4000, the better, as the 0x36xx range is currently being assigned officially.

Reply:
@drashna Vortex applied this VID from WB, thanks.